jsapp/js/components/library/libraryUtils.es6

```diff
@@ -28,3 +28,27 @@ export function isOnPublicCollectionsRoute() {
     path.split('/')[2] === 'public-collections'
   );
 }
+
+/**
+ * Checks if on `/library/asset/…`
+ */
+export function isOnSingleCollectionRoute() {
```

Comment:
Just a minor thing - this function name should be rather `isOnLibraryAssetRoute`, as it will work the same regardless if asset type is "collection", "question" or "template" :)